### PR TITLE
Production release

### PR DESCRIPTION
**Pull request by eoinkelly**

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -17,6 +17,32 @@ class Application < Rails::Application
     config.app_protocol = ENV['APP_PROTOCOL']
     config.base_url = "#{config.app_protocol}://#{config.app_domain_name}/"
 
+    ##
```
Comment (louise-r-blue) [inline config/application.rb:20]: Great documentation. Just a thought. Would it be of any benefit to have this referred to in the PR description as is really the essence of the accumulative work toward this PR?

**Review by louise-r-blue — APPROVED** [review]

**Review by Br3nda — APPROVED** [review]

**Comment by Br3nda** [comment]
Can we hold off relasing this until monday? so we know someone is around if it break.

**Comment by Br3nda** [comment]
Just saw @eoinkelly 's message on slack. Seems okay to release if you're going to be around - but that means no beeroclock for you

**Comment by eoinkelly** [comment]
Releasing now. I'll babysit it to make sure it lands ok

**Pull request merged**